c182s.xml

```diff
@@ -758,8 +758,8 @@
                 </domain>
                 <range>
                     <!-- reducing factor: 1=no reduction/0=full blocked; min=no-ice, max=full-ice -->
```

Comment:
@Zaretto Are you sure this is correct?
The min/max output is meant to be a inflow reducing factor, thus, when ice is maximized, the reducing factor has to decrease to model "less air gets inside".

Ice >= 0.010 inches should yield in 0.75 reduction of air original air inflow.
With your fix, we get: Ice==0 yield 0.75 original inflow.